Microsoft.Toolkit.Uwp.SampleApp/Controls/AadAuthControl.xaml.cs

```diff
@@ -73,7 +86,7 @@ private void AadAuthControl_Loading(FrameworkElement sender, object args)
                     // re-auth required, it might fail, so store it in a local variable to avoid impacts to previous controls
                     _scopesForReAuth = _scopes.Concat(adminPerms).Distinct().ToArray();
 
-                    var result = _graphService.Logout().Result;
+                    var result = await _graphService.Logout();
```

Comment:
This seems to have exposed a concurrency issue. Repro steps:
1. Start sample app and navigate to the AadLogin sample. Login!
2. Navigate to the PlannerTaskList sample. App will crash inside AadLogin.cs, line 47. 

Looks like the issue is with MicrosoftGraphService.IsAuthenticated in the MicrosoftGraphService.Logout() method.

Reply:
I changed the order to set IsAuthenticated to false first.